app/src/main/java/org/simple/clinic/patient/sync/PatientPayload.kt

```diff
@@ -75,7 +80,9 @@ data class PatientPayload(
         updatedAt = updatedAt,
         deletedAt = deletedAt,
         recordedAt = recordedAt,
-        syncStatus = newStatus)
+        syncStatus = newStatus,
+        reminderConsent = reminderConsent ?: Granted
```

Comment:
Can you add a `TODO` here to make this non-nullable when the reminder consent property is pushed to production?